test/cmdlineTests/yul_optimizer_steps_with_empty_cleanup_sequence/input.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+pragma abicoder v2;
+
+contract C {
+    constructor() payable {
+        assembly ("memory-safe") {
+            let a := 0
+            revert(0, a)
```

Comment:
Since the intention of this may not be completely clear, we could consider adding some comments like:

```suggestion
            // Without the cleanup sequence this will not be simplified to ``revert(a, a)``.
            revert(0, a)
```
here and analogously
```
// The cleanup sequence will simplify this to ``revert(a,a)``
```
in the other cases.

But also fine without such comments.

Reply:
Makes sense, will add these comments.

Reply:
I think it would make more sense to put the comment above the line, not below it :)